apps/polls/assets/QuestionForm.jsx

```diff
@@ -0,0 +1,135 @@
+var React = require('react')
+var django = require('django')
+var FlipMove = require('react-flip-move')
+var ChoiceForm = require('./ChoiceForm')
+
+let QuestionForm = React.createClass({
+  getInitialState: function () {
+    return {}
+  },
+
+  /*
+  |--------------------------------------------------------------------------
+  | Choice state related handlers
+  |--------------------------------------------------------------------------
+  */
+
+  handleUpdateChoiceLabel: function (index, label) {
+    this.props.updateChoiceLabel(this.props.index, index, label)
+  },
+  handleDeleteChoice: function (index) {
+    this.props.deleteChoice(this.props.index, index)
+  },
+
+  handleAppendChoice: function () {
+    this.props.appendChoice(this.props.index)
+  },
+
+  /*
+  |--------------------------------------------------------------------------
+  | Question state related handlers
+  |--------------------------------------------------------------------------
+  */
+
+  handleDelete: function () {
+    this.props.deleteQuestion(this.props.index)
+  },
+
+  handleMoveUp: function () {
+    this.props.moveQuestionUp(this.props.index)
+  },
+
+  handleMoveDown: function () {
+    this.props.moveQuestionDown(this.props.index)
+  },
+
+  handleLabelChange: function (e) {
+    var index = this.props.index
+    var label = e.target.value
+    this.props.updateQuestionLabel(index, label)
+  },
+
+  render: function () {
+    return (
+      <section>
+        <div className="commenting">
+          <div className="commenting__content">
+            <div className="form-group">
+              <label
+                htmlFor={'id_questions-' + this.props.key + '-name'}>
+                {django.gettext('Question:')}
+              </label>
+              <input
+                className="form-control"
```

Comment:
`.form-control` is of no importance anymore, we removed it. I know, theoretically the same goes for documents.